classes/renderers/coursework_renderer.php

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file is empty? is it needed?

Reply:
I opened a ticket CTP-3714 to remove this and the other empty file